src/markdown.rs

```diff
@@ -52,8 +56,8 @@ impl Renderer {
                         languages.insert(content.to_string());
                     }
                 },
-                Event::Start(Tag::Image(_, url, _)) if url.starts_with("./") => {
-                    *url = format!("file://{}/{}", root_dir.display(), &url[2..]).into();
+                Event::Start(Tag::Image(_, url, _)) if !re_absolute_url.is_match(url) => {
+                    *url = format!("file://{}/{}", root_dir.display(), re_path_prefix.replace(url, "")).into();
```

Comment:
This treats any source paths that don't start with `<protocol>://` as relative to the location of the readme file. In the future, a path starting as `/` could be treated as relative to the root of the repo instead. I'll open an issue for it when this is merged.